tests/test_java.py

```diff
@@ -1164,3 +1164,19 @@ def test_including_method_modifiers(
             assert is_function_in_collapsed("private static Fibonacci.fibonacci(I)J_[j]", collapsed)
         else:
             assert not is_function_in_collapsed("private static Fibonacci.fibonacci(I)J_[j]", collapsed)
+
+
+@pytest.mark.parametrize("java_line_numbers", ["none", "line-of-function"])
+@pytest.mark.parametrize("in_container", [True])
+def test_including_line_numbers(
+    application_pid: int,
+    profiler_state: ProfilerState,
+    java_line_numbers: str,
+) -> None:
+    with make_java_profiler(profiler_state, java_line_numbers=java_line_numbers) as profiler:
+        collapsed = snapshot_pid_collapsed(profiler, application_pid)
+        if java_line_numbers == "line-of-function":
+            assert is_function_in_collapsed("Fibonacci.fibonacci:9(I)J_[j]", collapsed)
+        elif java_line_numbers == "none":
+            assert is_function_in_collapsed("Fibonacci.fibonacci(I)J_[j]", collapsed)
+            assert not is_function_in_collapsed("Fibonacci.fibonacci:9(I)J_[j]", collapsed)
```

Comment:
```suggestion
        else:
            assert java_line_numbers == "none"
            assert is_function_in_collapsed("Fibonacci.fibonacci(I)J_[j]", collapsed)
            assert not is_function_in_collapsed("Fibonacci.fibonacci:9(I)J_[j]", collapsed)
```
Not expecting anything else are you? :)